app/services/feature_flag_service/feature_flags_api.md

```diff
@@ -0,0 +1,223 @@
+
+# feature_flags/v1/community
+
+## GET /:community_id
+
+Request body: empty
+
+Response body:
+
+```ruby
+{ community_id: 123
+, features: 
+  [ :topbar_v1
+  , :new_login
+  ]
+}
+```
+
+## POST /:community_id
+
+Request body:
+
+```ruby
+[ :some_feature
+, :some_other_feature
+]
+```
+
+Response body:
+
+```ruby
+{ community_id: 123
+, features: 
+  [ :topbar_v1
+  , :new_login
+  , :some_feature
+  , :some_other_feature
+  ]
+}
+```
+
+## DELETE /:community_id
+
+Request body:
+
+```ruby
+[ :topbar_v1
+, :some_feature
+]
+```
+
+Response body:
+
+```ruby
+{ community_id: 123
+, features: 
+  [ :new_login
+  , :some_other_feature
+  ]
+}
+```
+
+## GET /enabled/:community_id/:feature
+
+Request body: empty
+
+Response body:
+
+```ruby
+{ enabled: true
+}
+```
+
+# feature_flags/v1/user
+
+## GET /:user_id
+
+Request body: empty
+
+Response body:
+
+```ruby
+{ user_id: 123
```

Comment:
Original intention here was maybe to describe a HTTP API that we could achieve by building a controller layer on top of this service. But from API driven perspective maybe it would be better to use the actual param names.